ietf/static_src/css/streamfield.scss

```diff
@@ -75,6 +75,11 @@ $border-colour: #bebebe;
         > tbody {
             width: 100%;
             background-color: $white;
+
+            img {
+                height: auto;
+                max-width: 100%;
+            }
```

Comment:
That will only fix it inside tables. Does this issue exist when you put an image in the WYSIWYG but not in a table?

Reply:
Just checked now, looks like images generally play nice outside of tables